solr/bin/solr.cmd

```diff
@@ -1187,8 +1187,8 @@ IF "%ENABLE_REMOTE_JMX_OPTS%"=="true" (
   set REMOTE_JMX_OPTS=
 )
 
-REM Enable java security manager (limiting filesystem access and other things)
-IF "%SOLR_SECURITY_MANAGER_ENABLED%"=="true" (
+REM Enable java security manager by default (limiting filesystem access and other things)
+IF NOT DEFINED SOLR_SECURITY_MANAGER_ENABLED (
```

Comment:
Sorry, maybe my comment was confusing, this logic is not correct. we need two "IF".
We don't need to change anything about the existing logic, but we need to introduce a new IF above it to set the default. something like this:
```
REM Enable by default (if the variable is not set by the user, we set it to true)
IF NOT DEFINED SOLR_SECURITY_MANAGER_ENABLED(
  set SOLR_SECURITY_MANAGER_ENABLED=true
)
...
REM Existing logic in master (Unchanged)
IF "%SOLR_SECURITY_MANAGER_ENABLED%"=="true" (
 ...
```